source/main.cc

```diff
@@ -365,6 +365,28 @@ parse_parameters (const std::string &input_as_string,
 
 
 /**
+ * Print help text
+ */
+void print_help()
+{
+  std::cout << "Usage: ./aspect [args] <parameter_file.prm>   (to read from an input file)"
+            << std::endl
+            << "    or ./aspect [args] --                     (to read from stdin)"
+            << std::endl
+            << std::endl;
+  std::cout << "    optional arguments [args]:"
+            << std::endl
+            << "       --version              (for information about library versions)"
+            << std::endl
+            << "       --help                 (for this usage help)"
+            << std::endl
+            << "       --print-xml-schema     (print xml parameters to standard output and exit)"
```

Comment:
> Oh, why stdout?

My thought is that this is more flexible. This way the caller can decide on the location and name.

> Also, is the word "schema" well understood?
I think so: https://en.wikipedia.org/wiki/Database_schema
I don't want to use something like "parameter-in-xml-format" because this might confuse people in thinking that we can load .xml files (we can not and are not planning on doing so).